Resources/views/CRUD/Association/list_collection.html.twig

```diff
@@ -0,0 +1,50 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+
+{% extends admin.getTemplate('base_list_field') %}
+
+{% block field %}
+    {% set route_name = field_description.options.route.name %}
+    {% set route_role = route_name|upper %}
```

Comment:
> Use Admin::getAccess()

Isn't it more `Admin::hasAccess()` ?
So it would be just
```twig
{% if field_description.associationAdmin.hasAccess(route_name, value) %}
```
?